libraries/cli/include/cli/config_jsons/mainnet.json

```diff
@@ -211,12 +211,12 @@
         "dpos": {
           "delegation_delay": "0x5",
           "delegation_locking_period": "0x5",
```

Comment:
to change this, we need to verify if we calculate avg block time correctly.

now we define it as 2.5 * lambda so for testnet where lambda = 1500m it should be 5.25s but in reality based on explorer it is 3-4 seconds